CHANGES.rst

```diff
@@ -1,3 +1,6 @@
+0.13.4 (Unreleased)
+===================
+
 0.13.3 (2019-05-04)
```

Comment:
D'oh! Just noticed now that the 0.13.3 release date is wrong. Should be 2019-0**6**-04.

Reply:
Is it June already? I'll fix it.